lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -318,7 +318,7 @@ def memory_replacement_map
 
     def options_replacement_map
       {
-        '@machine_options@' => @machine_options.join(','),
+        '@machine_options@' => @machine_options.concat(device_machine_options).join(','),
```

Comment:
Surprisingly `Array#concat` is a destructive operation, which means `@machine_options` will be modified. `+` operator should be used instead to avoid such a side effect.

Reply:
done